index.html

```diff
@@ -367,6 +223,50 @@ <h2>About me</h2>
             <img src="./images/IconCancel.png" alt="Close menu">
         </button>
     </div>
+    <section class="pop-up-main d-flex menu-hidden" id="pop-up">
+        <div class="float-window d-flex">
+            <button class="close-pop-up">
+                <img src="images/IconCancel.png" alt="Close pop-up window">
+            </button>
+            <h2 class="pop-up-title">Multi-Post Stories</h2>
+            <ul class="d-flex">
+                <li>
+                    <a href="#">
+                        <img src="images/htmlTag.png" alt="htmlButton">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="images/bootstrapTag.png" alt="bootstrapButton">
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="images/rubyTag.png" alt="rubyButton">
+                    </a>
+                </li>
+            </ul>     
+            <div class="d-flex second-div-pop-up">
+                <img class="pop-up-img" src="images/yogaDesktop.png" alt="Project's Thumbnail">
+                <div class="third-div-pop-up d-flex">
+                    <p class="pop-up-text">Lorem Ipsum is simply dummy text of the printing 
+                        and typesetting industry. Lorem Ipsum has been 
+                        the industry's standard dummy text ever since 
+                        the 1500s, when an unknown printer took a galley 
+                        of type and scrambled it to make a type specimen book.
+                        It has survived not only five centuries, but also the
+                        leap into electronic typesetting, remaining essent
+                        Lorem Ipsum is simply dummy text of the printing 
+                        and typesetting industry. 
+                    </p>
```

Comment:
- [ ] this information should come from JavaScript array object, let follow the instructions.
- [ ] The pop-up should fit properly on the screen, I was unable to see all the pop-up content. See below

![popup](https://user-images.githubusercontent.com/53124548/136144549-a6e849da-923f-40ce-9a26-10b05e01b796.png)